backend/tests/test_views.py

```diff
@@ -516,3 +516,40 @@ def test_deleting_ticket():
     assert md.Ticket.objects.all()[0].ticketid == ticketid2
 
     resetDB()
+
+
+@pytest.mark.django_db
+def test_updating_columns_order():
+    """
+    Test updating columns order
+    Test that columns new order is correct
+    """
+
+    api_client = APIClient()
+
+    boardid = addBoard(uuid.uuid4()).boardid
+    columnid1 = addColumn(boardid, uuid.uuid4()).columnid
+    columnid2 = addColumn(boardid, uuid.uuid4()).columnid
+    columnid3 = addColumn(boardid, uuid.uuid4()).columnid
+    columnid4 = addColumn(boardid, uuid.uuid4(), "Swimlane", True).columnid
+
+    data = [
+        {"columnid": str(columnid3)},
+        {"columnid": str(columnid4)},
+        {"columnid": str(columnid2)},
+        {"columnid": str(columnid1)},
+    ]
+
+    response = api_client.put(
+        reverse("columns_on_board", args=[boardid]), data=json.dumps(data), content_type="application/json"
+    )
+    assert response.status_code == 200
+
```

Comment:
It actually might be a good idea to have a separate test that the columns get ordered specifically in the database by checking the ordernum.

For example:
```suggestion

    assert md.Column.objects.get(pk=columnid1).ordernum == 3
    assert md.Column.objects.get(pk=columnid2).ordernum == 2
    assert md.Column.objects.get(pk=columnid3).ordernum == 0
    assert md.Column.objects.get(pk=columnid4).ordernum == 1

```


But I think this still lgtm